CONTRIBUTING.md

```diff
@@ -43,6 +43,14 @@ Don't forget to add your new article to the `SUMMARY.md` to let it be rendered t
 
 Please make `Draft Pull requests` early so we can follow your progress and can give early feedback (see the following section).
 
+## Style guide
+
```

Comment:
Would propose a small introduction text here why we apply a style guide to the docs

Reply:
YES! I talked about this in the issue

Reply:
done in https://github.com/rust-unofficial/patterns/pull/201/commits/777a0f73e56263987f3392536be217d934ac7ed7
feel free to suggest changes